rootpy/stl.py

```diff
@@ -156,6 +156,13 @@
     # try to create it. See https://github.com/rootpy/rootpy/issues/328
     mkdir_p(DICTS_PATH)
 
+include_list = os.path.join(userdata.BINARY_PATH, 'include_paths.list')
+log.debug('Using %s to get additional include paths' % include_list)
+if os.path.exists( include_list ):
+    with open(include_list) as inc_list:
+        for line in inc_list:
+            log.debug('adding %s to the include paths' % line.strip())
```

Comment:
place ``line = line.strip()`` above this and then use ``line`` in both lines below. Also use new-style string formatting here.